src/starter-kit wiki page folder/_Sidebar.md

```diff
@@ -0,0 +1,32 @@
+**[Home](https://github.com/bids-standard/bids-starter-kit/wiki)**

```

Comment:
I think we can delete this file: if really needed we will pull some of its content from the history of the repo.

Reply:
Sure, will do.

Reply:
Already made the change. Thank you.